numba/cloudpickle/cloudpickle.py

```diff
@@ -203,6 +203,10 @@ def _lookup_module_and_qualname(obj, name=None):
 
     module_name = _whichmodule(obj, name)
 
+    if module_name not in sys.modules:
```

Comment:
Why isn't this case cover by the check at line 221?

https://github.com/numba/numba/blob/abffa7060eedff66cfa3e7b3876018c1f22e5e0e/numba/cloudpickle/cloudpickle.py#L220-L227

Reply:
You're right. I've moved the delete obj.__module__ into this block now